tools/create-jira-release

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env node
+
+const { program } = require('commander')
+const fetch = require('node-fetch')
+const OAuth = require('oauth').OAuth
+
+const JIRA_URL = 'https://issues.integreat-app.de'
+
+program
+  .option('-d, --debug', 'enable extreme logging')
+  .requiredOption('--project-name <project-name>', 'the name of the jira project, e.g. integreat-webapp')
+  .requiredOption('--access-token <access-token>', 'version name of the new release')
+  .requiredOption('--private-key <privateKey>')
+  .requiredOption('--consumer-key <consumer-key>')
+
+const initializeOAuth = (privateKeyBase64, consumerKey) => {
+  const privateKey = Buffer.from(privateKeyBase64, 'base64').toString('ascii')
+
+  return new OAuth(
+    '',
+    '',
+    consumerKey,
+    privateKey,
+    '1.0',
+    '',
+    'RSA-SHA1'
+  )
+}
+
+const postToJira = async (path, body, accessToken, privateKeyBase64, consumerKey) => {
+  const consumer = initializeOAuth(privateKeyBase64, consumerKey)
+
+  await new Promise((resolve, reject) => consumer.post(
+    `${JIRA_URL}${path}`,
+    accessToken,
+    '',
+    JSON.stringify(body),
+    'application/json',
+    (error, response) => error ? reject(error) : resolve(response)
+  ))
+}
+
+const putToJira = async (path, body, accessToken, privateKeyBase64, consumerKey) => {
+  const consumer = initializeOAuth(privateKeyBase64, consumerKey)
+
+  await new Promise((resolve, reject) => consumer.put(
+    `${JIRA_URL}${path}`,
+    accessToken,
+    '',
+    JSON.stringify(body),
+    'application/json',
+    (error, response) => error ? reject(error) : resolve(response)
+  ))
+}
+
+const getFromJira = async path => {
+  const response = await fetch(`${JIRA_URL}${path}`)
+  return response.json()
+}
+
+const createRelease = async ({ newVersionName, accessToken, privateKey, consumerKey, projectName }) => {
+  const date = new Date()
+  const newReleaseDate = `${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}`
+
+  const projects = await getFromJira('/rest/api/2/project')
+  const project = projects.find(project => project.name === projectName)
+  const projectKey = project.key
+
+  // Create a new jira version
```

Comment:
Should probably be
```suggestion
  // Create a new version in jira
```

Reply:
Will change in https://github.com/Integreat/integreat-react-native-app/pull/292